js/plot/util/scales.js

```diff
@@ -21,7 +21,9 @@ var _ = require('lodash');
 var commonbolus = require('./commonbolus');
 
 var scales = function(opts) {
-  opts = opts || {};
+  opts = _.assign({}, opts) || {};
+
+  var GLUCOSE_MM = 18.01559;
```

Comment:
feels wrong but i guess there is no other way. We have the variable ``GLUCOSE_MM`` twice  that I have seen. Can we have it once and parsed in? Just incase for some dumb reason one is different to the other and we can't easily detect via the tests